.gitignore

```diff
@@ -9,4 +9,4 @@ docs/
 scratchpad/
 .direnv/
 .idea/
-.env
+.env*
```

Comment:
Is this okay to add. I'm using 1Password for key management with `.env.1password` file.